### ISSUE TITLE
internationalize new CSV strings

### ISSUE THREAD

**Issue by huss** [OpenEnergyDashboard/OED#703]
### Is your feature request related to a problem? Please describe.

The CSV upload feature has a large number of strings that are returned to the user. They are all in English. They should be internationalized as is the case for other messages for user.

### Describe the solution you'd like

The strings are believed to reside in these files:

- src/server/routes/csv.js
- src/server/services/csvPipeline/uploadReadings.js
- src/server/services/pipeline-in-progress/loadCsvInput.js
- src/server/services/pipeline-in-progress/loadArrayInput.js
- src/server/services/pipeline-in-progress/processData.js
- src/server/services/csvPipeline/uploadMeters.js
- src/server/services/csvPipeline/success.js
- src/server/services/csvPipeline/saveCsv.js

### Describe alternatives you've considered

For now, the user only sees English.

### Additional context

None.

**Comment by huss** [comment]
Thanks to @KatherineVickstrom for helping look into this as a quick issue to address. That helped me realize that we have never done translation on the server side so something new seems needed. What is not yet clear but quick thoughts are:

- Get react-intl to work on the server but it does not currently use React. A web search indicated it might be possible outside React but it will mean new code/effort.
- Pass back strings with the data.js translation key to then replace on the client side. Another option might be passing back the string with an error code or the translation key for substitution. (Okay this really needs more thought.)

I'm open to ideas on how to do this as the string are returned to the user. I'm just trying to record the issue.

**Comment by davidNicolas-cecs** [comment]
We are working on this! (Jesse Garcia & Edwin Kofler), and will message Steve shortly.

**Comment by huss** [comment]
PR #866 worked on this. We need to decide the direction to go in a future release.

Update 7/24: Since it has now sat for a while, I am closing the PR until decisions are made. We may want to open it back up when someone is ready to work on this.

**Comment by skhaterina** [comment]
Can i work on this?

**Comment by huss** [comment]
> Can i work on this?

@skhaterina Yes, you are welcome to work on this. As the issue indicates, there has been previous work and discussion to consider. Please let me know if we can help (but I'm not expert here).

**Comment by skhaterina** [comment]
@huss  Hi! I'm working on CSV internationalization for OED and have run into a blocker with the development environment setup. After following the standard installation process (docker compose up) on a fresh clone from the main/development branch, the database initialization fails with ERROR: permission denied to create extension "btree_gist". The database creates the oed user and databases successfully, but npm run createdb silently fails during schema creation - no tables are created, and I can't log in or access the application. I've tried multiple branches (main, development, v1.0.0), complete Docker cleanups, and fresh clones, all with the same result. I successfully worked on OED in September 2024 when it was functioning properly, so this appears to be a recent regression. Could you point me to a currently stable branch/commit for development work, or let me know if there's a known workaround for this issue? I'm on macOS with Docker Desktop. Any guidance would be greatly appreciated - I'm eager to contribute but blocked on environment setup. Thanks!

**Comment by huss** [comment]
This is is again available for people to work on. Please see the PR for efforts already done.
